packages/manager/fragments/navbar/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@ovh-ux/manager-fragment-navbar",
```

Comment:
**thought**: A fragment generator could be provided to generate basic config in a next step